src/lib/pipeline.ml

```diff
@@ -7,15 +7,16 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
 
   module Stdlib = Biokepi.EDSL.Library.Make(Bfx)
 
-
   let to_bam_dna ~parameters ~reference_build samples =
     let sample_to_bam sample =
       let open Biokepi.EDSL.Library.Input in
       let bam =
         match sample with
         | Bam {bam_sample_name; path; how; sorting; reference_build} ->
-          Bfx.bam ?sorting ~sample_name:bam_sample_name
-            ~reference_build (Bfx.input_url path)
+          (Bfx.bam ?sorting ~sample_name:bam_sample_name
+             ~reference_build (Bfx.input_url path))
+          |> Bfx.picard_reorder_sam
+            ?mem_param:parameters.Parameters.picard_java_max_heap
```

Comment:
Not something I'm sold on, but 1. because the pipeline will fail if it's not ordered by the reference we use 2. because I don't want another CLI switch for this 3. this is the "rarer" case where we don't realign, so if it takes a *little* bit longer this way (relative to the total pipeline time), :shrug:.